docs/jade-getting-started.md

```diff
@@ -125,12 +125,14 @@ Unfortunately, some manual configuration is also necessary:
 export VAULT_ADDR=https://clotho.broadinstitute.org:8200
 
 # configure helm
-helm repo add stable
 helm repo add datarepo-helm https://broadinstitute.github.io/datarepo-helm
 helm plugin install https://github.com/thomastaylor312/helm-namespace
 helm plugin install https://github.com/databus23/helm-diff
 helm repo update
 
+# launch docker desktop - this installs docker in /usr/local/bin
+open -a docker
+
```

Comment:
Do we need to call this before helm since helm sometimes does stuff with kubectl?

Reply:
When I ran these instructions, the first step that needed `docker` was `gcloud auth configure-docker`. But I can move it up if you'd prefer it to be first in the list of manual steps.